### ISSUE TITLE
-j0 hides duplicate-code

### ISSUE THREAD

**Issue by The-Compiler** [PyCQA/pylint#987]
When I have two files like this:

``` python
def fun1():
    pass

def fun2():
    pass

def fun3():
    pass
```

And I run pylint (1.5.0 or current master) without `-j`, I get `duplicate-code` as expected:

```
$ pylint 1.py 2.py -rn --disable=all --enable=duplicate-code    
No config file found, using default configuration
************* Module 2
R:  1, 0: Similar lines in 2 files
[...]
```

Adding `-j0` hides the error:

```
$ pylint 1.py 2.py -rn --disable=all --enable=duplicate-code -j0
No config file found, using default configuration

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 9.17/10, +0.83)
```

I figured this out because I got the message on AppVeyor but not on Travis, so this might be a Linux-only issue?

**Comment by doublethefish** [comment]
Fixed by PR #3458, also see #3314 .
